x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/esql/styles.tsx

```diff
@@ -14,3 +15,10 @@ export const EmbeddedDiscoverContainer = styled.div`
   display: grid,
   place-items: center
 `;
+
+// TODO remember to remove the className added to discover/public/components/discover_grid_flyout/discover_grid_flyout.tsx when removing this
+export const TimelineESQLGlobalStyles = createGlobalStyle`
+  body:has(.timeline-portal-overlay-mask) .DiscoverFlyout {
```

Comment:
Thanks! I noticed it was `.euiFlyout` before and was wondering how that affected the other flyouts 😅 . Nice work, tested and works great!